src/Shell/PluginShell.php

```diff
@@ -22,13 +22,15 @@
  */
 class PluginShell extends Shell
 {
-
     /**
-     * Contains tasks to load and instantiate
+     * Tasks to load
      *
      * @var array
      */
-    public $tasks = ['Assets'];
+    public $tasks = [
```

Comment:
You are suggested to merged your changes, not override. Don't remove `Assets` from the task list.